readme.rst

```diff
@@ -12,6 +12,8 @@ You can either create a standalone documentation project based on this repositor
 
 **Note:** We're planning to provide the contents of this repository as an installable package in the future, but currently, you need to copy and update the required files manually.
 
+The next two sections provide manual steps for enabling the starter pack for your repository. See :doc:`setup-script` for an automated method (currently in beta).
```

Comment:
Hmm, this link doesn't work on GitHub. We could link to the documentation output instead? Neither way is really nice ... but I feel since this here is the readme, we should make sure users can read it on GitHub.

Reply:
I did an absolute link to the output page - but since this PR is not merged yet it doesn't exist yet. Not ideal but can't think of a better way.